nest-city-account/Dockerfile

```diff
@@ -29,6 +36,15 @@ CMD [ "npm", "run", "start:debug" ]
 
 # production
 FROM app-base AS prod
+# When copying from build, it copies symlink in node_modules to ../../openapi-clients and expects the original directory
+# to be present.
+# TODO: Find a way how to use --install-links flag in npm install to avoid this.
+RUN mkdir -p /home/node/openapi-clients && chown -R node:node /home/node/openapi-clients
+
+WORKDIR /home/node/openapi-clients
+COPY --chown=node:node --from=openapi-clients /app/ ./
```

Comment:
(optional) Why copy it in the prod from the image and also in build? Shouldn't just once be enough? If you need it in the build step can't you just copy it from there then?

Reply:
Explained in the comment below, I improved the comment at leastä
https://github.com/bratislava/konto.bratislava.sk/pull/2353/commits/bab606371294c773f3e12d5842be05727f067442